idd/Energy+.idd.in

```diff
@@ -4502,6 +4503,20 @@ Schedule:Constant,
        \type real
        \default 0
 
+Schedule:File:Shading,
+       \min-fields 1
+       \memo A Schedule:File:Shading points to a text computer file that has
```

Comment:
Probably want to reword this, don't need to say "computer file".  And instead of text file, how about CSV file?